src/Workspaces/Core/Portable/Workspace/Solution/Solution.cs

```diff
@@ -191,22 +191,16 @@ private static Project CreateProject(ProjectId projectId, Solution solution)
         /// <summary>
         /// Gets a document or a source generated document in this solution with the specified document ID.
         /// </summary>
-        internal async ValueTask<Document?> GetDocumentAsync(DocumentId? documentId, bool includeSourceGenerated = false, CancellationToken cancellationToken = default)
+        internal ValueTask<Document?> GetDocumentAsync(DocumentId? documentId, bool includeSourceGenerated = false, CancellationToken cancellationToken = default)
         {
             var project = GetProject(documentId?.ProjectId);
             if (project == null)
             {
-                return null;
+                return default;
```

Comment:
That's actually safe?

Reply:
I think so. Why wouldn't be?

Reply:
https://github.com/dotnet/runtime/blob/79ae74f5ca5c8a6fe3a48935e85bd7374959c570/src/libraries/System.Threading.Tasks.Extensions/tests/ValueTaskTests.cs#L36